src/screens/WebView/HelpCenter.js

```diff
@@ -18,7 +19,7 @@ const HelpCenter = ({navigation}) => (
       style={styles.webview}
       renderLoading={() => (
         <View style={styles.loading}>
-          <ActivityIndicator size="large" color="#00ADB5" />
+          <ActivityIndicator size="large" color={COLORS.holyTosca} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing hardcoded color values with references to color constants from a theme file is a good practice. This will make it easier to manage and update color schemes in the future.
```diff
- <ActivityIndicator size="large" color="#00ADB5" />
+ <ActivityIndicator size="large" color={COLORS.holyTosca} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->